src/entry.js

```diff
@@ -22,6 +22,8 @@ module.exports = (argv) => {
     process.exit(1);
   }
 
+  require('update-notifier')({ pkg: require('../package.json') }).notify();
```

Comment:
Is notifier smart enough to not shoot off warnings while we are developing the lib?